<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubmitDialog</class>
 <widget class="QDialog" name="SubmitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基准点位置</string>
  </property>
  <widget class="QWidget" name="widget_1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>911</width>
     <height>591</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QToolButton" name="toolButton_previous">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 104, 186);</string>
    </property>
    <property name="text">
     <string>上一步</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>881</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_handin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 186, 37);</string>
    </property>
    <property name="text">
     <string>确认提交</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>881</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 244, 172);
border-color: rgb(234, 244, 172);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>   请确认你提交的信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
